packages/design-tokens/src/scss/generated/_gradients.scss

```diff
@@ -1,12 +1,12 @@
 // Generated Gradients from Supernova. Do not edit manually.
-$gradient-background-basic-ltr: linear-gradient(90deg, #fff 0%, #ffffff00 100%) !default;
-$gradient-background-basic-rtl: linear-gradient(270deg, #fff 0%, #ffffff00 100%) !default;
-$gradient-diagonal: linear-gradient(135deg, #073256 0%, #185c95 100%) !default;
-$gradient-horizontal: linear-gradient(90deg, #073256 0%, #185c95 100%) !default;
+$gradient-diagonal: linear-gradient(var(--angle, 135deg), #073256 0%, #185c95 100%) !default;
+$gradient-gradient-background-basic-ltr: linear-gradient(var(--angle, 90deg), #fff 0%, #ffffff00 100%) !default;
```

Comment:
Ouch, I think that something is missing and something is extra here… 🙈 

1. Why is the prefix `gradient` duplicated? This should have been renamed in the Supernova. 
2. Isn't there missing [the new gradient token from Spirit](https://github.com/lmc-eu/spirit-design-system/pull/822/files#diff-096a4db89bb37d62ce6f62c848d584715b76b8d636974e38e0fb342b09d6f7e3R2) `$gradient-background-basic-overlay`? This token could be used instead of `$gradient-gradient-background-basic-ltr` and `$gradient-gradient-background-basic-rtl`, so these two tokens could be deleted and the new token could be directly used in the CSS file with the CSS custom property to define angle ( https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/ScrollView/_ScrollView.scss#L118).
3. `$gradient-diagonal` and `$gradient-horizontal` could be replaced as well with the one token – eg. `$gradient-background-primary-overlay` and the rest could handle `--angle` custom property.

Please, let's talk about this PR on our synchro call, and if you see it the same way, let's fix it. 🙏🏻